rsp_jupyter_extensions/handlers/ghostwriter.py

```diff
@@ -0,0 +1,43 @@
+"""Ghostwriter handler, used for redirection bank shots once you've
+started a new lab."""
+from jupyter_server.base.handlers import JupyterHandler
+
+
+class Ghostwriter_handler(JupyterHandler):
```

Comment:
```suggestion
class GhostwriterHandler(JupyterHandler):
```

Perhaps PascalCase naming for consistency

Reply:
I think the old capitalization was in there from the JL 0.x days.